src/main/java/net/snowflake/client/core/SFSessionProperty.java

```diff
@@ -85,7 +85,10 @@ public enum SFSessionProperty {
   DISABLE_GCS_DEFAULT_CREDENTIALS("disableGcsDefaultCredentials", false, Boolean.class),
 
   JDBC_ARROW_TREAT_DECIMAL_AS_INT("JDBC_ARROW_TREAT_DECIMAL_AS_INT", false, Boolean.class),
-  DISABLE_SAML_URL_CHECK("disableSamlURLCheck", false, Boolean.class);
+
+  DISABLE_SAML_URL_CHECK("disableSamlURLCheck", false, Boolean.class),
+
+  JDBC_USE_HARDCODED_TIMEZONE("JDBC_USE_HARDCODED_TIMEZONE", false, Boolean.class);
```

Comment:
is this flag to change the used timezone or applying the formatter? because the logic base on that flag seems to change different places? maybe the flag name is misleading now?

Reply:
It should be used to determine whether a hardcoded value should be used for the formatter (previous behavior) or to take what the user sets JDBC_FORMAT_DATE_WITH_TIMEZONE to. It's essentially there for backwards compatibility to keep existing behavior unless the user changes JDBC_USE_HARDCODED_TIMEZONE

Reply:
I can definitely rename it. Aside from your suggestion of JDBC_FORMAT_DATE_WITH_DEFAULT_TIMEZONE, maybe JDBC_DEFAULT_FORMAT_DATE_WITH_TIMEZONE, since the default isn't the timezone, it's the true/false value for format date. Which would you prefer?

Reply:
the name `JDBC_DEFAULT_FORMAT_DATE_WITH_TIMEZONE ` is fine - let's use it